src/main/resources/application.properties

```diff
@@ -1,6 +1,15 @@
 spring.application.name=karuna_backend
 spring.profiles.active=@active.profile@
 
+#prefix for each endpoint
+server.servlet.contextPath=/api
+
+#Log sql queries
+spring.jpa.show-sql=true
```

Comment:
Need to think if we realy want to see this logs on production

Reply:
You are right, I should add this to dev.properties
Thank you!